src/client/proxy/handlers/common.ts

```diff
@@ -110,8 +111,8 @@ const getHttpRequestHandler = () => async (req: FastifyRequest, res: FastifyRepl
                 rootDir = deployConfig.rootDir;
             }
 
-            const publicPath = path.resolve(zappDir, rootDir);
-            const routesJsonPath = path.resolve(zappDir, 'routes.json');
+            const publicPath = path.resolve(zappDir, rootDir); if (!existsSync(publicPath)) throw new Error('Public path ' + publicPath + ' doesnt exist.');
```

Comment:
Purely stylish note, but anyway: I'm not sure why eslint doesn't show warning that the string is too long
We discussed some time ago that we should avoid `if` without braces (and also there are line breaks missing)

Reply:
You can split the assignment from if here

Can you link to the discussion that we should avoid ifs without braces? Our code base is becoming a behemoth (this file is 471 lines) and I'm trying to compact it while refactoring, so splitting simple ifs into 3 lines where 1 would suffice seems like a waste

Reply:
Sorry, I can't find the thread, but I remember we discusse it with @tankakatan and the team, and agreed on this.
This idea for writing 3 statements in 1 line still seems not good for code readabiliti IMO. And if we want to make files shorter, we could rather split the code into several files.